plugins/connectrpc/query-es/v2.0.2/buf.plugin.yaml

```diff
@@ -0,0 +1,22 @@
+version: v1
+name: buf.build/connectrpc/query-es
+plugin_version: v2.0.2
```

Comment:
Wait, why are you changing this without updating the source?

https://github.com/connectrpc/connect-query-es/releases/tag/v2.0.1

Reply:
Because we needed different options for the plugin but the source is identical.

Reply:
This is precisely why we have revisions. So you can make config-related changes without modifying anything about the upstream version.

The version you bumped should match whatever is published upstream.

In this case just make the config changes and it'll bump the revision.

Reply:
Ah so my original instinct was correct and i regret my follow up.

Reply:
Hehe, what I'm trying to say is your initial commit was correct. It modified the config and that's it, the rest is handles for you, so there's nothing to dig into.

Reply:
Yep, all good. Pushed up my original change.